### DISCUSSION TITLE
What are the key considerations when using semantic HTML tags?

### DISCUSSION THREAD

**Discussion by Sbiswas001** [Sbiswas001/Student-Portfolio#30] [Q&A] [answered]
A semantic element clearly describes its meaning to both the browser and the developer.

Examples of non-semantic elements: <div> and <span> - Tells nothing about its content.

Examples of semantic elements: <form>, <table>, and <article> - Clearly defines its content.

**Comment by itsyourap** [comment, marked as answer]
> What are the key considerations when using semantic HTML tags? 

- Semantic HTML improves accessibility by providing meaning and context to web content, which aids screen readers and other assistive technologies in understanding and navigating the page.
- Search engines use semantic HTML to better understand the content and context of a web page, which can improve search rankings.
- Semantic HTML makes the code more readable and easier to maintain, as the purpose of each section of the document is clear.
- Developers can more easily understand the structure and function of the code, facilitating updates and modifications.
- Semantic tags provide better hooks for CSS and JavaScript, making it easier to apply styles and behaviors based on the role of elements rather than their generic tags.
